utilities/rpp-unittests/HIP_NEW/Tensor_hip_pkd3.cpp

```diff
@@ -2364,7 +2364,7 @@ int main(int argc, char **argv)
         count += dstDescPtr->strides.nStride;
 
```

Comment:
coding of many stuff in this file is not done very good. Are we going to keep this file

Reply:
This file is to be deleted once the new test_suite is ready, @snehaa8 We need to remove any updates to these files.

Reply:
Yeah Abishek, missed removing this during cleanup. I will clean this up.